client/lib/wp/node.js

```diff
@@ -2,5 +2,12 @@
  * Internal dependencies
  */
 var wpcom = require( 'lib/wpcom-undocumented' );
+var config = require( 'config' );
 
-module.exports = wpcom( require( 'wpcom-xhr-request' ) );
+wpcom = wpcom( require( 'wpcom-xhr-request' ) );
+
+if ( config.isEnabled( 'support-user' ) ) {
+	wpcom = require( 'lib/wp/support' )( wpcom );
+}
+
+module.exports = wpcom;
```

Comment:
These changes are required to give the unit tests access to the `wpcomSupport` functions